### PR TITLE
Task rubric

### PR DESCRIPTION
**Pull request by tddumont**
This branch contains the side navigation bar and the updated Task Rubric drawer component, which is now connected to the backend.

### CODE REVIEW

**Review by ConnorBoulais — APPROVED** [review]
Overall looks good, nice job

File: src/Components/GoalsScreen.js [hunk 1]
```diff
@@ -167,8 +148,46 @@ function GoalsScreen() {
...
+    const [subName, setSubName] = useState('');
+    const [subDate, setSubDate] = useState('');
+
+    function handleAddSubgoal() {
+      if (subName != '' && subDate != '') {
+        setSubgoals([...subgoals, {id: subgoals.length, name: subName, done: false, goal: 'n/a', duedate: subDate}]);
+        setSubName('');
+        setSubDate('');
+      }
+    }
```
Comment (samiur98) [inline src/Components/GoalsScreen.js:168]: Looks good.

**Review by samiur98 — APPROVED** [review]
A lot of commits but should be able to be merged in.

**Comment by tddumont** [comment]
This merge contains the side navigation bar and the updated Task Rubric drawer component, which is now connected to the backend.

**Pull request merged**